apps/cowswap-frontend/src/modules/swap/containers/SwapWidget/index.tsx

```diff
@@ -188,21 +186,27 @@ export function SwapWidget({ topContent, bottomContent }: SwapWidgetProps) {
   }
   const showTwapSuggestionBanner = !enabledTradeTypes || enabledTradeTypes.includes(TradeType.ADVANCED)
 
-  const swapWarningsTopProps: SwapWarningsTopProps = {
-    chainId,
-    trade,
-    showTwapSuggestionBanner,
-    buyingFiatAmount,
-    priceImpact: priceImpactParams.priceImpact,
-    tradeUrlParams,
-  }
+  const swapWarningsTopProps: SwapWarningsTopProps = useMemo(
```

Comment:
This object is used through spread operator, so no actually need to wrap in in memo.

```
<SwapWarningsTop {...swapWarningsTopProps} />
<SwapWarningsBottom {...swapWarningsBottomProps} />
```

Reply:
The linter was complaining about it 🤷